GliaWidgets/Sources/LockIsolated/LockIsolated.swift

```diff
@@ -0,0 +1,100 @@
+import Foundation
+
+/// A generic wrapper for isolating a mutable value with a lock.
+@dynamicMemberLookup
+final class LockIsolated<Value>: @unchecked Sendable {
```

Comment:
The indentation is wrong on this file

Reply:
Fixed fc7a7400893c7c46ab9eab3df3a22340a5df1b88.